docs/Fetch-SSB.md

```diff
@@ -0,0 +1,198 @@
+
+# config
+
+All ssb options here: <https://github.com/ssbc/ssb-config#configuration>
+
+## default config
+
+Agregore has this default configuration:
+
+```js
+  ssbOptions: {
+    appname: 'agregore-ssb',
+    ssbd: {
+      runServer: true,
+      plugins: require('@metacentre/shipyard-ssb')
+    }
+  },
+```
+
+The ssb data directory resides at `~/.agregore-ssb`
```

Comment:
Is there a way we could place the data inside a `ssb` folder within the Agregore data directory beside everything else?

Reply:
That was the first thing I tried and it was HARD. ssb really expects data in the home directory even though it uses rc under the hood. I think perhaps the ssb-config module is opinionated on this. I could have another go, but it may be a hair tearing exercise haha

Reply:
That's okay we can just keep it as is for the first iteration. 🤷